libs/net/dal/Config/S3Options.cs

```diff
@@ -0,0 +1,95 @@
+using Amazon.Runtime;
+using Amazon.S3;
+
+namespace TNO.DAL.Config;
+/// <summary>
+/// S3 config
+/// </summary>
+public static class S3Options
```

Comment:
Generally using static classes this way isn't ideal.  It makes it difficult or impossible to use dependency injection and then later unit tests.

Is there a reason you cannot use the default configuration options like everything else?  Then dependency injection works out of the box correctly.

If you need a helper class, then it should also not be a static class.  If you only need a single instance of a shared helper class then configure dependency injection so that it only uses that one class.

Reply:
Sure. let me follow other options styles.